### DISCUSSION TITLE
Encountered a GitHub design bug on a project's wiki page.

### DISCUSSION THREAD

**Discussion by mdibnjihan** [community/community#109716] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, recently I am working on customizing my Polybar installation on Linux. I headed over to the project's wiki page hosted on GitHub. I got confused for a very long time as I could not find what I was looking due to this **BUG**. Here is a detailed preview of what happened.
I am browsing the wiki on my fresh Arch installation.
![Polybar wiki page](https://github.com/community/community/assets/101347202/f2fe3ad3-d99d-462a-9038-3c693f56733d)
An screenshot of the page I am currently visiting.
If you look at the right side there the documentation has `Pages (35)` section. But, if you look closer then you will notice that the **dopdown icon is showing the opposite feedback to the user from what it was supposed to do**. For this reason I mistook that the Pages section was already open and skipped a whole lot section of the Wiki.
Here are some selective screenshot of my experience.
Screenshot of the behaviour.
![menu_closed](https://github.com/community/community/assets/101347202/eeedb663-a438-45c2-b936-1f2075e383b6)
Here the menu is closed. But, it seems to be open.
![menu_open](https://github.com/community/community/assets/101347202/68f86e41-fd91-4b52-a390-f4e2683f8c89)
Here the menu is open.
This could cause confusion among users like me who are new to GitHub or kind of not familiar with GitHub products that much. 

System Information
```
Operating System: Arch Linux x86_64
Device model: Dell Latitude e4310
Kernel: 6.7.5-arch1-1
```

Browser information
```
Mozilla Firefox 123.0 64 bit edition for GNU/Linux
```

**Comment by namith1003** [comment]
It seems this link https://support.github.com/contact/bug-report is the only one remaining where you can report a bug that you have encountered, maybe you can file it here and see if they respond to it.

Hope this answers your question.

**Reply by mdibnjihan** [reply]
Thanks.
